website/templates/index.mako

```diff
@@ -1,324 +1,11 @@
-          </div>
-        </div>
-
-        <div class="row hidden-xs hidden-sm">
-          <div class="col-md-7 col-md-offset-2">
-            <h3>Version control makes life easier</h3>
-            <p>The OSF makes version control effortless. My PI, my lab mates, and I have access to previous versions of a file at any time&#151;and the most current version is always readily available.<br/><small><em>Erica Baranski, PhD Student, Social and Personality Psychology Funder Lab, UC Riverside</em></small></em></small></p>
-          </div>
-          <div class="col-md-3">
-            <img src="/static/img/front-page/user3.jpg" class="img-circle img-responsive" alt="Erica Baranski" />
-          </div>
-        </div>
-
-        <div class="row hidden-xs hidden-sm">
-          <div class="col-md-3">
-            <img src="/static/img/front-page/user4.jpg" class="img-circle img-responsive" alt="Anne Allison" />
-          </div>
-          <div class="col-md-7">
-            <h3>A centralized hub of information</h3>
-            <p>The OSF creates a centralized hub of information where I can oversee a diversity of research projects across multiple classes. The centralization, organization and anywhere-access save me the time and energy necessary for managing these projects.<br/><small><em>Anne Allison, Associate Professor of Biology at Piedmont Virginia Community College</em></small></em></small></p>
-          </div>
-        </div>
-
-
-      </div>
-    </div>
-    <div class="space-top space-bottom feature-6">
-      <div class="container">
-        <div class="row">
-          <div class="col-md-8">
-            <h2><strong>Free and open source.</strong></h2>
-            <h4>The OSF is a public good built to support your research.</h4>
-            <a href="#" class="btn btn-info btn-lg">Get started</a>
-          </div>
-          <div class="col-md-4 hidden-xs hidden-sm">
-            <div id="logo">
-              <div class="circle" id="circle-1"><span></span></div>
-              <div class="circle" id="circle-2"><span></span></div>
-              <div class="circle" id="circle-3"><span></span></div>
-              <div class="circle" id="circle-4"><span></span></div>
-              <div class="circle" id="circle-5"><span></span></div>
-              <div class="circle" id="circle-6"><span></span></div>
-              <div class="circle" id="circle-7"><span></span></div>
-              <div class="circle" id="circle-8"><span></span></div>
-            </div>
-          </div>
-        </div>
-      </div>
-
-    </div>
-
-
+<%def name="logged_in(name)">
+    %if name != '':
+        <%include file="home.mako"/>
+    %else:
+        <%include file="landing.mako"/>
+    %endif
 </%def>
 
-<%def name="footer()">
+${logged_in(user_name)}
```

Comment:
> For clarification, when you are asking these questions, are you expecting a response from me, or is this a record of what you're thinking when you review?

This is a record of what I'm thinking when I review. You can chime in if you want but you don't have to. If I'm not able to figure something out on my own I'll put it on a punchlist, and when I make a punchlist that's when I'm definitely expecting a response from you. However, this PR is small and tight enough that we may not need a punchlist. :-)